docker/s6/services/sidecar.svc/run

```diff
@@ -17,12 +17,17 @@ if [[ -n "$SIDECAR_LISTENER" ]]; then
 	sed -i.bak 's~^urls *= *\[~urls = \[ "'"$SIDECAR_LISTENER"'", ~' sidecar.toml
 fi
 
+# Update the bind_ip if set as an env var to override the original
+if [[ -n "$BIND_IP" ]]; then
```

Comment:
Yes, this is the only part that I thought wasn't working still and was what I was chatting with you about yesterday.

Thanks for finishing that.